README.md

```diff
@@ -1 +1,64 @@
-# Nobelisti
\ No newline at end of file
+# Nobelisti
+
+## Instalation
+
+0. Ensure you have installed (docker)[https://docs.docker.com/engine/installation/] and (docker-compose)[https://docs.docker.com/compose/install/]
+```
+$ docker --version
+Docker version 17.06.2-ce, build cec0b72
+$ docker-compose --version
+docker-compose version 1.16.1, build 6d1ac21
+```
+
+1. Clone the project
+```
+git clone git@github.com:TIS2017/Nobelisti.git
+```
+
+2. Build and run the container
+```
+docker-compose up --build
+```
+
+3. Get docker's IP and save it to /etc/hosts
+```
+echo $(docker network inspect bridge | grep Gateway | grep -o -E '[0-9\.]+')
+```
+
+Open `/etc/hosts` and append `IP.ADDRESS.FROM.ECHO    symfony.dev`.
+
+
+4. Change permissions so symfony can save cache, logs & sessions.
+```
+sudo chmod -R 777 src/var/cache src/var/logs src/var/sessions
```

Comment:
does not work on OSX

Reply:
I'll try to find a better solution for this.